pkg/template/helper.go

```diff
@@ -56,11 +56,11 @@ func executeTestCombination(v VersionTestTemplate) error {
 	return nil
 }
 
-// AddTestCases returns the test case based on the name to be used as customflag.
-func AddTestCases(names []string) ([]testCase, error) {
-	var testCases []testCase
+// TestCases returns the test case based on the name to be used as customflag.
+func TestCases(names []string) ([]testCase, error) {
+	var tcs []testCase
 
-	testCase := map[string]testCase{
+	tc := map[string]testCase{
```

Comment:
same testCase/testcase

Reply:
it will be a shadow, the map already has this name